scoreboard.py

```diff
@@ -167,35 +167,35 @@ def finalsScoreboard():
     rep += "<tr>"
     rep += "<td>"
     leagueDev = League(league_id=1020397, year=year)
-    teamDrew = leagueDev.teams[3]
-    lineupDrew = [3117251, 15847, 4242335, 4360310, 2577327, 4036378, 2576414,4428331,-16013]
-    rep += printTeam(teamDrew, lineupDrew, week)
+    teamAbe = leagueDev.teams[1]
```

Comment:
Since Abe's team id is 8.
```suggestion
    teamAbe = leagueDev.teams[7]
```